includes/class-servers-list.php

```diff
@@ -51,7 +51,7 @@ function ( $value, $key ) use ( &$hidden ) {
 
     public function prepare_items() {
         if ( ! class_exists( 'Predis\Client' ) ) {
-            require_once dirname( __FILE__ ) . '/predis/autoload.php';
+            require_once WP_REDIS_PLUGIN_PATH . '/vendor/predis/predis/autoload.php';
```

Comment:
Should we commit the `vendor` folder to the repo with this PR?

Reply:
You are right, we obviously need include the `vendor/predis` directory!

Reply:
Resolved in https://github.com/rhubarbgroup/redis-cache/pull/186/commits/300d9dc6d9fd370e15c7ac6e688d2983cc879883